modules/EnergyManagement/EEBUS/manifest.yaml

```
@@ -0,0 +1,71 @@
+description: EEBUS module
+config:
+  manage_eebus_grpc_api_binary:
+    description: Whether the module should manage the eebus grpc api binary
+    type: boolean
+    default: true
+  eebus_service_port:
+    description: Port for the control service, this will be sent in the SetConfig call.
+    type: integer
+    default: 4715
+  grpc_port:
+    description: Port for grpc control service connection. This is the port on which we will create our control service channel and start the grpc binary with.
+    type: integer
+    default: 50051
+  eebus_ems_ski:
+    description: EEBUS EMS SKI
+    type: string
+    required: true
+  certificate_path:
+    description: Path to the certificate file used by eebus go client. If relative will be prefixed with everest prefix + etc/everest/certs. Otherwise absolute file path is used.
+    type: string
+    required: false
+    default: eebus/evse_cert
+  private_key_path:
+    description: Path to the private key file used by eebus go client. If relative will be prefixed with everest prefix + etc/everest/certs. Otherwise absolute file path is used.
+    type: string
+    required: false
+    default: eebus/evse_key
+  eebus_grpc_api_binary_path:
+    description: Path to the eebus grpc api binary. If relative will be prefixed with everest prefix + libexec. Otherwise absolute file path is used.
+    type: string
+    required: false
+    default: eebus_grpc_api
+  vendor_code:
+    description: Vendor code for the configuration of the control service
+    type: string
+    required: true
+  device_brand:
+    description: Device brand for the configuration of the control service
+    type: string
+    required: true
+  device_model:
+    description: Device model for the configuration of the control service
+    type: string
+    required: true
+  serial_number:
+    description: Serial number for the configuration of the control service
+    type: string
+    required: true
+  failsafe_control_limit:
```

```suggestion
  failsafe_control_limit_W:
```
Adding the unit as suffix could help to avoid mistakes in code